docs/stories/03-inputs/Combobox.stories.tsx

```diff
@@ -456,10 +514,11 @@ export const ComboboxProps = {
       },
     },
     creatable: {
-      control: 'boolean',
-      description: 'If true, allows creating new options',
+      control: 'radio',
+      options: [false, true, 'visible'],
+      description: 'If true, allows creating new options. If "visible", always shows the create option.',
       table: {
-        type: { summary: 'boolean' },
+        type: { summary: 'boolean | "visible"' },
```

Comment:
I'm afraid that “visible” is understood as the basic visible state. This could lead to the belief that if it's not set to “visible”, then it's “invisible”. Don't you think “sticky” would be more appropriate?